blackbox/docs/cli.txt

```diff
@@ -1,9 +1,9 @@
 .. highlight:: sh
 .. _cli:
 
-===========
-CrateDB CLI
-===========
+====================
+The CrateDB CLI Tool
```

Comment:
I don't think `tool` is the right thing as this is basically the `CrateDB` binary. I would call it `The CrateDB CLI options` or something like that.